modules/documents/client/views/partials/modal-document-confirm-publish.html

```diff
@@ -3,10 +3,12 @@
 		<button class="btn btn-default close" type="button" ng-click="confirmDlg.cancel()">
 			<span aria-hidden="true">&times;</span>
 		</button>
-		<h3 class="modal-title">Confirm Publish File(s)</h3>
+		<h3 ng-if="confirmDlg.publishableFiles.length === 0">No files can be published</h3>
```

Comment:
Providing the code was not needed.  I know what a filter is but the idea of moving public facing text out of the markup and replacing it with more lines of code behind the scene feels wrong. Nevertheless if this is what is desired then fine.  Can you now describe which other ng-if's you want to see changed we can get this done in one go.